sunsynk-power-flow-card/sunsynk-power-flow.js

```diff
@@ -312,70 +312,70 @@ class SunsynkPowerFlowCard extends LitElement {
-            <circle id="pv2-dot" cx="0" cy="0" r="3" fill="${stateObj8.state === '0' ? 'transparent' : '#ff9933'}">
+            <path id="pv2-line" d="M 135 92 L 135.02 117 Q 135.03 127 128.02 127 L 121 127" class="${config.solar_installed === 'no' ? 'st12' : ''}" fill="none" stroke="#ff9933" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="2 2" pointer-events="stroke"/> 
+            <circle id="pv2-dot" cx="0" cy="0" r="3" class="${config.solar_installed === 'no' ? 'st12' : ''}" fill="${stateObj8.state === '0' ? 'transparent' : '#ff9933'}">
               <animateMotion dur="9s" repeatCount="indefinite" keyPoints="0;1" keyTimes="0;1" calcMode="linear">
                 <mpath xlink:href="#pv2-line"/>
               </animateMotion>
             </circle>
 
-            <rect x="51" y="112" width="70" height="30" rx="4.5" ry="4.5" fill="none" stroke="#ff9933" pointer-events="all"/>
+            <rect x="51" y="112" width="70" height="30" rx="4.5" ry="4.5" fill="none" stroke="#ff9933" pointer-events="all" class="${config.solar_installed === 'no' ? 'st12' : ''}"/>
             <rect x="279" y="135" width="70" height="70" rx="10.5" ry="10.5" fill="none" stroke="#558fc1" pointer-events="all"/>
             <rect x="382" y="243" width="70" height="30" rx="4.5" ry="4.5" fill="none" stroke="#558fc1" pointer-events="all"/>
             <rect x="278" y="54" width="70" height="30" rx="4.5" ry="4.5" fill="none" stroke="#5fb5ab" pointer-events="all"/>
             <rect x="169.15" y="145" width="70" height="50" rx="7.5" ry="7.5" fill="none" stroke="#959595" pointer-events="all"/>
-            <rect x="0" y="54" width="70" height="30" rx="4.5" ry="4.5" fill="none" stroke="#ff9933" pointer-events="all"/>
-            <rect x="101" y="54" width="70" height="30" rx="4.5" ry="4.5" fill="none" stroke="#ff9933" pointer-events="all"/>
-            <rect x="0" y="295" width="70" height="70" rx="10.5" ry="10.5" fill="none" stroke="#f3b1c9" pointer-events="all"/>
+            <rect x="0" y="54" width="70" height="30" rx="4.5" ry="4.5" fill="none" stroke="#ff9933" pointer-events="all" class="${config.solar_installed === 'no' ? 'st12' : ''}"/> 
+            <rect x="101" y="54" width="70" height="30" rx="4.5" ry="4.5" fill="none" stroke="#ff9933" pointer-events="all" class="${config.solar_installed === 'no' ? 'st12' : ''}"/>
+            <rect x="0" y="284.75" width="70" height="70" rx="10.5" ry="10.5" fill="none" stroke="#f3b1c9" pointer-events="all"/>
 
-            <path id="bat-line" d="M 179 259.87 L 105.03 259.89 Q 95.03 259.9 95.03 269.86 L 95.03 290" fill="none" stroke="#f3b1c9" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="2 2" pointer-events="stroke"/>
+            <path id="bat-line" d="M 179 259.87 L 105.03 259.89 Q 95.03 259.9 95.03 269.86 L 95.03 279.83" fill="none" stroke="#f3b1c9" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="2 2" pointer-events="stroke"/>
```

Comment:
Please keep the path I extended the line since I moved things down
<path id="bat-line" d="M 179 259.87 L 105.03 259.89 Q 95.03 259.9 95.03 269.86 L 95.03 290" fill="none" stroke="#f3b1c9" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="2 2" pointer-events="stroke"/>